LayoutTests/platform/glib/imported/w3c/web-platform-tests/cookies/samesite/multiple-samesite-attributes.https-expected.txt

```diff
@@ -1,14 +1,42 @@
+CONSOLE MESSAGE: Fetch API cannot load https://www1.localhost:9443/cookies/resources/dropSameSiteMultiAttribute.py due to access control checks.
+Blocked access to external URL https://www1.localhost:9443/cookies/resources/redirectWithCORSHeaders.py?status=307&location=https%3A%2F%2F127.0.0.1%3A9443%2Fcookies%2Fresources%2FimgIfMatch.py%3Fname%3Dsamesite_unsupported_none%26value%3D0.10277761549984665
+Blocked access to external URL https://www1.localhost:9443/cookies/resources/redirectWithCORSHeaders.py?status=307&location=https%3A%2F%2F127.0.0.1%3A9443%2Fcookies%2Fresources%2FimgIfMatch.py%3Fname%3Dsamesite_lax_none%26value%3D0.10277761549984665
+Blocked access to external URL https://www1.localhost:9443/cookies/resources/redirectWithCORSHeaders.py?status=307&location=https%3A%2F%2F127.0.0.1%3A9443%2Fcookies%2Fresources%2FimgIfMatch.py%3Fname%3Dsamesite_unsupported_strict%26value%3D0.10277761549984665
+Blocked access to external URL https://www1.localhost:9443/cookies/resources/redirectWithCORSHeaders.py?status=307&location=https%3A%2F%2F127.0.0.1%3A9443%2Fcookies%2Fresources%2FimgIfMatch.py%3Fname%3Dsamesite_lax_strict%26value%3D0.10277761549984665
+Blocked access to external URL https://www1.localhost:9443/cookies/resources/redirectWithCORSHeaders.py?status=307&location=https%3A%2F%2F127.0.0.1%3A9443%2Fcookies%2Fresources%2FimgIfMatch.py%3Fname%3Dsamesite_unsupported_lax%26value%3D0.10277761549984665
+Blocked access to external URL https://www1.localhost:9443/cookies/resources/redirectWithCORSHeaders.py?status=307&location=https%3A%2F%2F127.0.0.1%3A9443%2Fcookies%2Fresources%2FimgIfMatch.py%3Fname%3Dsamesite_strict_lax%26value%3D0.10277761549984665
+Blocked access to external URL https://www1.localhost:9443/cookies/resources/redirectWithCORSHeaders.py?status=307&location=https%3A%2F%2F127.0.0.1%3A9443%2Fcookies%2Fresources%2FimgIfMatch.py%3Fname%3Dsamesite_none_unsupported%26value%3D0.10277761549984665
+Blocked access to external URL https://www1.localhost:9443/cookies/resources/redirectWithCORSHeaders.py?status=307&location=https%3A%2F%2F127.0.0.1%3A9443%2Fcookies%2Fresources%2FimgIfMatch.py%3Fname%3Dsamesite_lax_unsupported%26value%3D0.10277761549984665
+Blocked access to external URL https://www1.localhost:9443/cookies/resources/redirectWithCORSHeaders.py?status=307&location=https%3A%2F%2F127.0.0.1%3A9443%2Fcookies%2Fresources%2FimgIfMatch.py%3Fname%3Dsamesite_strict_unsupported%26value%3D0.10277761549984665
+Blocked access to external URL https://www1.localhost:9443/cookies/resources/redirectWithCORSHeaders.py?status=307&location=https%3A%2F%2F127.0.0.1%3A9443%2Fcookies%2Fresources%2FimgIfMatch.py%3Fname%3Dsamesite_unsupported%26value%3D0.10277761549984665
 
 PASS Same-host images are strictly same-site
-FAIL Subdomain images are strictly same-site promise_test: Unhandled rejection with value: "'samesite_unsupported_none=0.9461741832459153' not present on https://www1.web-platform.test:9443"
```

Comment:
This stood out to me also and I'm not sure why, but I'll try to look at it in more detail.

FYI the GLib ports run these tests with web-platform.test typically. The macOS runners do not, so that is one reason the test results here may not align with wpt.fyi.